Automattic/wp-calypso
# Login: Implement login by site URL

Requires: D129616-code

Related to https://github.com/Automattic/wp-calypso/issues/83359
Fixes https://github.com/Automattic/wp-calypso/issues/83359

## Proposed Changes

This implements logging by using site address. The backend obtains the owner's email address and sends them a magic link. Then it shows a censored email to the requester. 

## Testing Instructions

1. Apply the patch.
2. Sandbox public api. 
3. Visit /log-in/site-url.
4. Type a site that is not owned by an a11n. 
5. You should be redirected to the success page and see a censored email address. 

## Next steps
1. Merge this since it is behind a flag.
2. Extend tests [here](https://github.com/Automattic/wp-calypso/blob/implement/login-by-site-url/client/state/login/magic-login/test/reducer.js#L23). 
3. Consult designers about copy and UX.
